vipps-login-from-phone-number-api-howitworks.md

```diff
@@ -14,44 +14,72 @@ END_METADATA -->
+ * Binding message: Include a binding message in the confirmation step for some extra assurance
+ * Delegated consents: In this extra step, merchants can collect the consents required to enroll the customer to your loyalty program
+ * Redirect: You may choose to redirect the user to a website of your choosing after login is completed
 
 ![login process](images/vipps-login-from-phone-number-process.png)
 
 The different steps are described and illustrated with screenshots below.
 
 ### 1. Initiate Vipps Login from phone number
 
-The merchant initiates Vipps Login based on phone number from the merchant's system. The user receives a push from Vipps on the phone and clicks the push to open Vipps.
+The merchant initiates Vipps Login on the users device from their system, based on the user's phone number.
+The user receives a push notification from Vipps and clicks the push to open the Vipps app.
 
-If the user has turned off push or for some other reason do not get push, they can open Vipps themselves and the process to confirm the login will start automatically.
+If the user has turned off push notifications or for some other reason do not get the push, they can open Vipps themselves and automatically start the login flow.
 
 ![User gets push message from Vipps](images/vipps-login-phone-push.png)
 
-### 2. Confirm in Vipps
+### 2. Confirmation
 
-After opening Vipps, the user will be asked to confirm sharing information with the merchant. The page will specify the information requested by the merchant.
+After opening Vipps, the user will be asked to confirm sharing information with the merchant. The first screen will specify the information requested by the merchant.
 
-The merchant can choose to show a confirmation code / binding message on the confirmation screen. This is for added security if the merchant would like their representatives or their customers to have an explicit check that it is the correct transaction that is being accepted.
+On this screen, the merchant can choose to show a binding message (confirmation code). This is for added security if the merchant would like their representatives or their customers to have an explicit check that it is the correct transaction that is being accepted.
 
-After confirming the login and sharing of information, the user will be asked for the formal consent to share information from Vipps to the merchant. This consent is only needed once per merchant and applies across all Vipps services. In other words, if the user has already consented to share the requested information with the merchant, no consent will be required.
+After confirming the login and sharing of information, the user will be asked for the formal consent to share information from Vipps to the merchant.
+This consent is only needed once per merchant and applies across all Vipps services. In other words, if the user has already consented to share the
```

Comment:
once per end-user?

Reply:
Text updates might be better to suggest in confluence, https://confluence.vipps.io/pages/viewpage.action?pageId=3067708835